promotion.md

```diff
@@ -1,6 +1,16 @@
 # Const promotion
 
-["(Implicit) Promotion"][promotion-rfc] is a mechanism that affects code like `&3`:
+"Promotion" is the act of guaranteeing that code not written in a const context
+(e.g. initalizer of a `const` or `static`, or an array length expression) will
```

Comment:
These are examples *for* code in const-context, right? Because to me this reads like examples for "code not written in a const context".